warehouse/internal/model/upload.go

```diff
@@ -33,10 +35,34 @@ const (
 	ColumnSizeError           JobErrorType = "column_size_error"
 	InsufficientResourceError JobErrorType = "insufficient_resource_error"
 	ConcurrentQueriesError    JobErrorType = "concurrent_queries_error"
-	UnknownError              JobErrorType = "unknown_error"
-	Noop                      JobErrorType = "noop"
 )
 
+var userFriendlyJobErrorCategoryMap = map[JobErrorType]string{
+	UnCategorizedError:        "Uncategorized error",
```

Comment:
[minor] Uncategorized is a single word
```suggestion
	UncategorizedError:        "Uncategorized error",
```